tutorials/3_transactions.md

```diff
@@ -83,14 +130,16 @@ Let's take a look at the artifacts generated by the tool. You should see:
   * `libverify/`: runtime libraries which define how require, assert, etc., are
     handled in each model.
 
-To instrument the model, we will need to modify `cmodel.c`. The code is not
-human-readable but we can fix this with `clang-format`:
+To understand the model, we will focus on `cmodel.c`. This file is unformatted,
+but we can improve its presentation with `clang-format`:
 
   * `mdkir build ; cd build`
   * `cmake .. -DSEA_PATH=/path/to/seahorn/bin`
   * `make run-clang-format`
```

Comment:
replace with (but check) `cmake --build . --target clang-format`

this will work independently of how build utility is called